packages/core-utils/src/common/hex-strings.ts

```diff
@@ -56,6 +56,12 @@ export const toHexString = (inp: Buffer | string | number | null): string => {
   }
 }
 
+/**
+ * Casts a number to a hex string without zero padding.
+ *
+ * @param n Number to cast to a hex string.
```

Comment:
number or bignumber

Reply:
Eh I think it's fine to leave as is, the type hints will show people that it's a number or big number